src/components v2/Atoms/Components/BrandGradientText/BrandGradientText.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <span ref="text" class="ph-brand-gradient-text">
+    <slot />
+  </span>
+</template>
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
+  name: 'P2BrandGradientText',
+});
+</script>
+
+<style lang="scss" scoped>
+.ph-brand-gradient-text {
+  display: inline-block;
+  background-clip: text;
+  -webkit-background-clip: text;
+  color: transparent;
+  background-image: var(--gradient-text-base-gradient);
```

Comment:
if you set this to `background:` it can also support color.

Reply:
Let's make sure that the integration actually swaps out brand gradient text for a typography component.

Reply:
> if you set this to `background:` it can also support color.

ah smart, I had to re-arrange the CSS for this to work, I assumed it only could use background-image